Release 0.4.1 (#46)
tarantool/metrics-export-role: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## Unreleased

### Fixed

> - Backward compatibility with Tarantool that hasn't version of metrics module with full Graphite support.

Reviewer: In fact, `metrics-export-role 0.4.0` is fully compatible with Tarantool 3.6 and older, since it requests `metrics 1.7.0` explicitly. In this patch, we make this dependency optional -- `metrics 1.7.0` (both built-in or external) is expected only in case ones uses `graphite` plugin, and it no longer installed as an external dependency.

Author: Edited description

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

> ## 0.4.1 - 2026-05-05

Reviewer: ```suggestion
## 0.4.1 - 2026-05-06
```

Author: Updated

Fixed backward compatibility with Tarantool versions where `metrics` module does not yet
provide full Graphite support.

### Fixed

> - Backward compatibility with Tarantool that hasn't version of metrics module with full Graphite support.

Reviewer 2: ```suggestion
- Backward compatibility with Tarantool versions, that don't have full Graphite support for the metrics module.
```

Author: Reformulated
